docs/release/release_0_4_3.md

```diff
@@ -0,0 +1,120 @@
+# napari 0.4.3
+
+We're happy to announce the release of napari 0.4.3!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+This release is the first release to have direct support for generating widgets from functions using [magicgui](https://napari.org/magicgui/) and the `viewer.window.add_functio_widget` method (#1856). We leverage the newly release `0.2` series of magicgui which seperates out an abstract function and widget API from its Qt backend. 
+
+In this release we also seperate out more of the Qt functionality from napari making it easier to run headless (#2039, #2055). We also add a `napari.run` method to then launch the napari application (#2056).
+
+We've also made good progress on our experimental support for an octree system for rendering large 2D multiscale images. You can try this functionality setting `NAPARI_ASYNC=1` as an environment variable.
```

Comment:
Is there not a separate `NAPARI_OCTREE` variable anymore?

Reply:
yup, replaced and included link to https://napari.org/docs/dev/events/rendering.html